modules/git/object_format.go

```diff
@@ -130,7 +130,6 @@ var (
 
 var SupportedObjectFormats = []ObjectFormat{
 	Sha1ObjectFormat,
-	Sha256ObjectFormat,
```

Comment:
I'm not a fan of many build tags.

> Is it conceivable to only support SHA256? or not worth the effort at this time?
Currently not, it can be another PR in future.